core/src/main/scala/kanaloa/reactive/dispatcher/queue/QueueProcessor.scala

```diff
@@ -21,6 +21,11 @@ class QueueProcessor(
   resultChecker:          ResultChecker
 ) extends Actor with ActorLogging with MessageScheduler {
 
+  val healthCheckSchedule = {
```

Comment:
But it's controlled. The attempt to recreate only happens per Healthcheck interval, while in the approach of recreate on RouteeFailed you will burn the cpu if the routee creation fails fast, right?